### ISSUE TITLE
noms-chat demos

### ISSUE THREAD

**Issue by gedw99** [attic-labs/noms#3808]
https://github.com/attic-labs/noms/blob/master/samples/go/decent/p2p-chat/main.go#L19

where is the IPFS wrapper ?

**Comment by gedw99** [comment]
My interest.

I am playing around with crdt, ipts.
And mom's is definitely in the same ball park.

Reason ?
Offline clients and synchronisation back to server (s).
Working on NAT and using QUIC. QUIC just got added to IPFS btw !

So I am just looking for samples to get going fast to experiment :)

**Comment by gedw99** [comment]
Btw samples could be in a different repo and vendor all their dependencies.
Will make it easier to hack on noms are not care about breaking samples

**Comment by aboodman** [comment]
Cool. You can just check out the code from before my change right now. I'll
figure out whether to undo this change to remove the IPFS store or fix the
demo soon.

Thanks!

**Comment by aboodman** [comment]
Yeah agree. I think someone else who was looking at this recently ran into
a problem that direction...

I think it was just how to make all the CLI tools know about the other
chunkstore type. Like if IPFS store is in another repo that is fine for
applications that directly link noms. But what about `noms show` and
friends?

**Comment by aboodman** [comment]
I tried to revert https://github.com/attic-labs/noms/commit/24dc7ad17000f04cf08179dc2973b0f24d13f525 but I can't seem to remember how to get gx to pull all the deps, so I still can't build :-/.
